### ISSUE TITLE
The loss keeps 100+ after 40epoches

### ISSUE THREAD

**Issue by 18875140516** [qqwweee/keras-yolo3#664]
I use this model to train o my own dataset to detect charecters. I use the pretrain model, the loss start from nearly 8000 to 100+ after 40epoches, and it can't decrease anymore. I have used K-means to calculate the new anchors, change the train data to the format the project said, but the loss still keep nearly 100. Does someone have the same problem with me?

**Comment by GiangHLe** [comment]
I can't help you with that, please share more information, for example:

+ Your hyperparameter
+ Some sample in your data
+ Your new anchors
+ How you divide train/test, number of sample
+  Loss and Val loss
+ etc

**Comment by 18875140516** [comment]
I use the model of this repository on "SCUT_FORU_DB_Release" dataset to detect the charactor. The dataset consist of the image which include various billboard with charactors, each charactor in the image has a box and a category in [a-z, A-Z, 0-9]. The rate of train/test is 9/1 which used in this model, the number of sample is 1162. Learning rate form 1e-3 at the beginning and decrease to 1e-4  with decay 0.01 after 40 epoches.  The loss is still nearly 100 and so is val_loss. I use the method in this repostory to calculate the anchor=[9,14,  11,24,  16,20,  21,29,  26,41,  37,47,  47,65,  69,97, 144,192]. Do you think whether yolov3 can do this job? I have a problem that if 'W' in a image but the half of this charactor is 'v' which must not tag as a box, how can model consider this situation?

**Comment by GiangHLe** [comment]
For the loss problems, depending on my experience:
+ Transfer learning from the weights of darknet which was trained by COCO. Warming up your model by freeze all layers except the last for 10 or 20 epochs (just like in this repo) then start to train all parameters.
+ Using bigger batch size 64-128.
+ Removing the learning rate decay, set the checkpoint so you can tune the hyperparameter and keep training from that point if you see the performance is not well (loss increase, loss unchanged, etc)
+ If you use Adam, start with lr = 1e-4, if SGD momentum, start with lr = 1e-2, momentum = 0.9. 


> Do you think whether yolov3 can do this job?

No. I used to try YOLO to recognize character, and its performance was not really well, this architecture is not suitable for that topic, I recommend you to use other deep learning models:
+ If you need to detect a sentence, CTPN is good for this. Github repo: https://github.com/eragonruan/text-detection-ctpn
+ If you only need to detect a word. EAST's performance is really well. Github repo:
https://github.com/argman/EAST.

**Comment by 18875140516** [comment]
I also try EAST to detect the character, but the my image need to be detect include only characters but not text or word. It like several numbers on the ruler. The model is really nice, but I think it not adapt to my problem. [https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1588046008837&di=a10113b6bc1dd03a915d489030849ace&imgtype=0&src=http%3A%2F%2Fe.hiphotos.baidu.com%2Fzhidao%2Fpic%2Fitem%2Fa6efce1b9d16fdfadb9de8e3b48f8c5495ee7bcd.jpg](url). The images are similay to this one, can you give me some advice? I now try create my own dataset, and use rpn + classification to test.

**Comment by GiangHLe** [comment]
@18875140516 That URL got some problems, I couldn't see the image, please upload your image to some other database (drive or google image). 

Recently, you are using rpn+classification, so it is something like Faster RCNN, right? If your data is not too complicated (only number), I think you shouldn't try to solve the problem by using a deeper, heavier, more complex model. Try to think about the simple way.

P/s: Next time, you should make your comment brighter, using enter or put the URL to the keyword, [like this](https://www.google.com/webhp?hl=vi&sa=X&ved=0ahUKEwjduISOv4rpAhW6xosBHdPRAs4QPAgH)
